tests/installation/ipxe_install.pm

```diff
@@ -48,7 +48,7 @@ sub poweron_host {
 sub set_pxe_boot {
     while (1) {
         my $stdout = ipmitool('chassis bootparam get 5');
-        last if $stdout =~ m/Boot Flag Valid[\d\D]*Force PXE/;
+        last if $stdout =~ m/Force PXE/;
```

Comment:
If you drop `Boot Flag Valid` from the regex, the function will accept a state where IPMI boot override is disabled and the SUT will boot according to BIOS settings, ignoring the `Force PXE` flag.

Reply:
To check `Boot Flag Valid` was added by me in a previous PR because we had a machine that only worked with this flag. We upgraded the BIOS and firmwire of this machine days ago, after that I found Boot Flag always remains 'Invalid' which stopped the test to be continued, fg. http://10.67.129.27/tests/28/logfile?filename=autoinst-log.txt

And it seems that the flag does not effect the tests anymore. I will do more verification runs on more machines.

Reply:
OK, never mind then.

Reply:
wait a moment... I might speak too early. it seems that something wrong with the machine, it boot from pxe to install SLE repeatedly...   I am going to figure out why its Boot Flag keeps 'Invalid' firstly.